From 448709125ce1010a29314c74795a56d3f25a10f7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 5 Dec 2024 13:54:12 +0800
Subject: [PATCH 175/273] AOSCOS: MIPS: loongson64: add missing includes for
 the loongson_system_configuration struct

Add missing includes for loongson.h and workarounds.h to fix build for
A1004/A1101/A1201/A1205 workaround code.

Fixes: "AOSCOS: MIPS: workaround A1004/A1101/A1201/A1205 display issues"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/loongson64/platform.c    | 2 ++
 arch/mips/loongson64/workarounds.c | 1 +
 2 files changed, 3 insertions(+)

diff --git a/arch/mips/loongson64/platform.c b/arch/mips/loongson64/platform.c
index eab6afbcad5c..f30205291ee6 100644
--- a/arch/mips/loongson64/platform.c
+++ b/arch/mips/loongson64/platform.c
@@ -10,6 +10,8 @@
 #include <linux/delay.h>
 #include <linux/platform_device.h>
 #include <loongson_hwmon.h>
+#include <loongson.h>
+#include <workarounds.h>
 
 /*
  * Kernel helper policy
diff --git a/arch/mips/loongson64/workarounds.c b/arch/mips/loongson64/workarounds.c
index 13b26d17b941..61709f6eef66 100644
--- a/arch/mips/loongson64/workarounds.c
+++ b/arch/mips/loongson64/workarounds.c
@@ -3,6 +3,7 @@
 #include <asm/bootinfo.h>
 #include <ec_wpce775l.h>
 #include <workarounds.h>
+#include <loongson.h>
 
 #define GPIO_LCD_CNTL		5
 #define GPIO_BACKLIGHIT_CNTL	7
-- 
2.47.1

